extensions/wikia/Helios/Helios.setup.php

```diff
@@ -35,3 +35,4 @@
 
 $wgHooks['UserSaveSettings'][] = 'Wikia\\Helios\\User::onUserSave';
 $wgHooks['UserSaveOptions'][] = 'Wikia\\Helios\\User::onUserSave';
+$wgHooks['UserLogout'][] = 'Wikia\\Helios\\User::clearAccessTokenCookie';
```

Comment:
Currently it is, although it seems the author's intention was to make a general purpose method.